templates/Wettbewerbe/eine_veranstaltung.html

```diff
@@ -21,4 +21,28 @@ <h2>Veranstaltung {{ veranstaltung.name }} ({{ veranstaltung.art }})</h2>
       <li><a style="color: gray;" href="mich_eintragen/">[meine Teilnahme eintragen]</a></li>
     {% endif %}
   </ul>
+  {% if veranstaltung.wettbewerbe.all %}
+  <p>Es sind folgende Wettbewerbe bei dieser Veranstaltung ausgetragen worden</p>
+  <ul>
+    {% for w in veranstaltung.wettbewerbe.all %}
+      <li><a href={{ w.get_absolute_url }}>{{ w.name }}</a>
+      {% if w.erfolge.all %}
+      <br />Folgende Erfolge wurden dabei erreicht
```

Comment:
Gibt's 'nen Grund, warum du nicht das ul zumachst? Würde auch das br ersparen, nich?

Reply:
das ul mache ich in Zeile 46 zu. Ist eine eingerückte Liste

Reply:
Ja, hast Recht, sorry!
Ich war irritiert dadurch, dass ich es perspektivisch anders machen will: dass wir alle Erfolge in eine Liste zusammenpappen (die man sortieren kann), nicht nach Wettbewerben geordnet. Aber stimmt, das ist ja noch anders, und ist für den Anfang auch gut.